### PR TITLE
Move and refactor allure report script

### PR DESCRIPTION
**Pull request by kneerose**
## List of changes

-  Move the allure report script file from base to tests/script
-  Update report script path on the backend and integration test
-  Update deprecated set-output to GITHUB_OUTPUT env

## Checklist

- [related issue](https://github.com/IntersectMBO/govtool/issues/)
- [x] My changes generate no new warnings
- [x] My code follows the [style guidelines](https://github.com/IntersectMBO/govtool/tree/main/docs/style-guides) of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the [changelog](https://github.com/IntersectMBO/govtool/blob/main/CHANGELOG.md)
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: tests/scripts/generate_latest_report_redirect.sh
Comment (mesudip) [inline tests/scripts/generate_latest_report_redirect.sh:1]: Let's move these files to .github/scripts instead

**Review by mesudip — CHANGES_REQUESTED** [review]

File: tests/scripts/generate_latest_report_redirect.sh
Comment (kneerose) [reply tests/scripts/generate_latest_report_redirect.sh:1]: done

**Comment by bosko-m** [comment]
@MSzalowski @Ryun1 Im not sure whether you noticed this one guys :)

**Comment by bosko-m** [comment]
@mesudip Are 4 reviewers mandatory?

**Review by mesudip — APPROVED** [review]
Looks Good

**Pull request merged**